backend/index.js

```diff
@@ -1,10 +1,86 @@
-const { PrismaClient } = require('@prisma/client')
-const prisma = new PrismaClient()
-const cors = require('cors')
-const express = require('express')
+// const { PrismaClient } = require('@prisma/client')
```

Comment:
As we discussed in chat, let's get rid of all the commented code before we merge to main :)